src/Behat/Behat/Definition/Printer/ConsoleDefinitionInformationPrinter.php

```diff
@@ -66,18 +91,16 @@ public function printDefinitions(Suite $suite, $definitions)
     /**
      * Extracts the formatted header from the definition.
      *
-     * @param Suite      $suite
-     * @param Definition $definition
-     *
      * @return string[]
      */
-    private function extractHeader(Suite $suite, Definition $definition)
+    private function extractHeader(?Suite $suite, Definition $definition): array
     {
         $pattern = $definition->getPattern();
         $lines = array();
+        $indent = $suite ? '{suite} <def_dimmed>|</def_dimmed> ' : '';
```

Comment:
If there is no suite, we don't indent the output text with the name of the suite